lib/SILGen/ManagedValue.h

```diff
@@ -270,6 +270,9 @@ class ManagedValue {
   /// Emit a copy of this value with independent ownership.
   ManagedValue copy(SILGenFunction &SGF, SILLocation loc) const;
 
+  /// Returns an unamanaged copy of this value.
+  ManagedValue unmanagedCopy(SILGenFunction &SGF, SILLocation loc) const;
```

Comment:
This needs a very large comment on it saying that it is the caller's responsibility to destroy the object/address and that we aren't creating cleanups for it.